### ISSUE TITLE
Voicing has trouble with Reading blocks when using touch

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/john-travoltage#409]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/645
When using touch on Win 10, it can be hard, or in some cases seeming impossible, to get the reading blocks to read. Even when they do read, they don't always seem to have the highlighting they do with mouse.

**Visuals**

https://user-images.githubusercontent.com/41024075/117711397-7027de00-b190-11eb-97ab-708ff844d761.MOV

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪John Travoltage‬
URL: https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-preferences.4/phet/john-travoltage_en_phet.html
Version: 1.6.0-preferences.4 2021-04-28 22:49:44 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36
Language: en-US
Window: 1280x658
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
There are two issues here. First, the touch areas for reading are quite small so it is difficult to activate them with a finger. Second, the reading highlight goes away on finger release.

To fix the first I will try to increase the touch area of the Reading Blocks, but we don't have a lot of space to work with. For the second, I think we need to separate the interactive highlight from the reading highlight, so that the yellow highlight persists as long as something is being spoken.

**Comment by jessegreenberg** [comment]
This was fixed as part of #412

**Comment by jessegreenberg** [comment]
This has been working as we want for some time now, closing.
